index.html

```diff
@@ -4,11 +4,11 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
+  
   <title>Hello Microverse</title>
 </head>
 <body>
-  <h1 class="header">Hello World!</h1>
+  <h1 class="header">Hello</h1>
```

Comment:
Please make sure your contents are wrapped in a main tag. because it makes your code looks more semantical